src/main/java/frc/robot/subsystems/TrapElvSubsystem.java

```diff
@@ -332,59 +336,67 @@ public void stowTrapElv() {
   }
 
   public Command zeroArm() {
-    return startEnd(
-            () -> {
-              // Command for zeroing elevator if elevator happens to be not at zero
-              // Runs elevator motors until there limit switches are pressed
-              if (!baseLimit.get()) {
-                baseMotor1.set(TrapElvConstants.ELV_ZEROING_SPEED);
-                baseMotor2.set(TrapElvConstants.ELV_ZEROING_SPEED);
-              } else {
+    if (isElv) {
+      return startEnd(
+              () -> {
+                // Command for zeroing elevator if elevator happens to be not at zero
+                // Runs elevator motors until there limit switches are pressed
+                if (!baseLimit.get()) {
+                  baseMotor1.set(TrapElvConstants.ELV_ZEROING_SPEED);
+                  baseMotor2.set(TrapElvConstants.ELV_ZEROING_SPEED);
+                } else {
+                  baseMotor1.stopMotor();
+                  baseMotor2.stopMotor();
+                  baseMotorOffset1 = baseMotor1.getEncoder().getPosition();
+                  baseMotorOffset2 = baseMotor2.getEncoder().getPosition();
+                }
+                if (!scoringLimit.get()) {
+                  scoringMotor.set(TrapElvConstants.ELV_ZEROING_SPEED);
+                } else {
+                  scoringMotor.stopMotor();
+                  scoringMotorOffset = scoringMotor.getEncoder().getPosition();
+                }
+              },
+              () -> {
                 baseMotor1.stopMotor();
                 baseMotor2.stopMotor();
-                baseMotorOffset1 = baseMotor1.getEncoder().getPosition();
-                baseMotorOffset2 = baseMotor2.getEncoder().getPosition();
-              }
-              if (!scoringLimit.get()) {
-                scoringMotor.set(TrapElvConstants.ELV_ZEROING_SPEED);
-              } else {
                 scoringMotor.stopMotor();
-                scoringMotorOffset = scoringMotor.getEncoder().getPosition();
-              }
-            },
-            () -> {
-              baseMotor1.stopMotor();
-              baseMotor2.stopMotor();
-              scoringMotor.stopMotor();
-            })
-        .withName("Zero Arm");
+              })
+          .withName("Zero Arm");
+    } else {
+      return run(() -> {});
```

Comment:
Trying to think through the pros and cons of returning an empty command vs disabling this in the robot.

Have we tested this in simulation?  Will a blank command cause any problems?  It is certainly an edge case.

Reply:
@ModBoyex tested this button in sim.  CLosing comment